section7Hash/recurringChar/recurringChar.go

```diff
@@ -0,0 +1,19 @@
+package recurringchar
+
+type recurringCharS1 struct{}
+
+func (s1 recurringCharS1) findRecurringChar(input1 []int) int {
+	kvPair := make(map[int]bool, 0)
```

Comment:
variable names should reflect its purpose like charSet etc

Reply:
changed to charSet